src/main/java/io/kamax/matrix/hs/_MatrixRoom.java

```diff
@@ -123,4 +126,27 @@ default void sendReadReceipt(String eventId) {
 
     _MatrixRoomMessageChunk getMessages(_MatrixRoomMessageChunkOptions options);
 
+    List<RoomTag> getUserTags();
+
+    void addUserTag(String tag);
```

Comment:
Are we sure we want to assume a default value for those? What's the impact?

If yes, it might be best to move the code into the interface itself using
```
default void addUserTag(String tag) {
  ...
};
```
Apply this comment to all others add methods.

Reply:
User tags are explicitely mentioned in the spec and it seems to me, they should be the default way to handle tags. That's why I created a special interface for them.
But reading the spec again, I just realized every other namespace is [valid too](https://matrix.org/speculator/spec/HEAD/client_server/unstable.html#id75) and could be used by clients to store special tags. So I probably should add a generic method which has the tag's name plus the namespace name as parameters. What do you think?

Reply:
The question was referring to `order` not being specified and therefore a default value would need to be provided by the implementations. So the questions really are:
- Are we sure we want to assume a default value for order?
- If yes, which one is the "good" default value? 0? 1? anything else?
- What's the impact of the default value?

If the answer is "yes" to assuming a default value, to ensure the value is coherent across all possible implementation, we might want to move the code that sets the default value to the interface itself, so it doesn't have to be implemented, but can be overwritten if needed.

Reply:
I was under the impression the order is mandatory. But reading your comments I'm not so sure any more. I will try out what happens, if I add tags in Synapse without setting the order.

Reply:
I have removed the default value for the order now.